.github/workflows/makefile_ci.yml

```
@@ -38,16 +38,34 @@ jobs:
 
       - name: Install dependencies
         run: make install
-      
+
       - name: Building CORE
         run: make build-core
-      
+
       - name: Run CORE tests
         run: make test-core
 
       - name: Run CLI tests
         run: make test-cli
 
+      - name: Create .env file
+        run: |
+          touch .env
+          echo "POSTGRES_HOST="$POSTGRES_HOST >> .env
+          echo "POSTGRES_PORT="$POSTGRES_PORT >> .env
+          echo "POSTGRES_USER="$POSTGRES_USER >> .env
+          echo "POSTGRES_PASSWORD="$POSTGRES_PASSWORD >> .env
+          echo "POSTGRES_DATABASE="$POSTGRES_DATABASE >> .env
+          echo "PORT="$PORT >> .env
+          echo "RUN_MIGRATIONS="$RUN_MIGRATIONS >> .env
+          cat .env
+
+      - name: Run postgres docker
+        run: docker-compose -f "docker-compose.dev.yml" up -d --build
+
+      - name: Checking container
+        run: docker ps -a
+
```

Be careful there, you need to remove the tests run
```suggestion
      - name: Run postgres docker
        run: docker-compose -f "docker-compose.dev.yml" up -d --build

      - name: Run Webapp unit tests
        run: make unit-test-webapp

      - name: Run Webapp e2e tests
        run: make e2e-test-webapp
        env:
          POSTGRES_HOST: 127.0.0.1
          POSTGRES_PORT: 5432
          POSTGRES_USER: postgres-kamon
          POSTGRES_PASSWORD: test
          POSTGRES_DATABASE: kamon_db
```

---

You can also use secret like in main branch
```
      env:
        SSH_USER: ${{ secrets.AWS_SSH_USER }}
        SSH_KEY: ${{ secrets.AWS_SSH_KEY }}
        SSH_HOST: ${{ secrets.AWS_SSH_HOST }}
```